### PR TITLE
Added draft cost values

### PR DESCRIPTION
**Pull request by oxade**
These are the draft costs and stubs with some marked todo because of measurement constraints and cross calibration required.
These new costs will not be enabled yet until clear deployment plan is rolled out.
Next steps:
1. Add snapshot testing to detect changes in TX costs.
2. Propose modifying VM call cost logic to suit our needs. Currently we cannot charge the way we want to
3. Set tighter bounds on constants
4. Move over to NativeContext

### CODE REVIEW

**Comment by oxade** [comment]
@tnowacki   @lxfind  @sblackshear @emmazzz 
hey folks I could use a stamp on this. It doesn't do much now but it's one of multiple parts before we cut over to new costing logic/values.

File: crates/sui-cost/tests/snapshot_tests.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```
Comment (tnowacki) [inline crates/sui-cost/tests/snapshot_tests.rs:3]: Should there be some code here?

**Review by tnowacki — APPROVED** [review]

File: crates/sui-cost/tests/snapshot_tests.rs (hunk 1 above)
Comment (lxfind) [reply crates/sui-cost/tests/snapshot_tests.rs:3]: Is there a plan to add code here?

Comment (oxade) [reply crates/sui-cost/tests/snapshot_tests.rs:3]: Yes, it's WIP

**Pull request merged**